src/layer/tile/GridLayer.js

```diff
@@ -156,14 +156,14 @@ L.GridLayer = L.Layer.extend({
 
 	_updateOpacity: function () {
 		if (!this._map) { return; }
-		var opacity = this.options.opacity;
 
 		// IE doesn't inherit filter opacity properly, so we're forced to set it on tiles
-		if (!L.Browser.ielt9 && !this._map._fadeAnimated) {
-			L.DomUtil.setOpacity(this._container, opacity);
+		if (L.Browser.ielt9 && !this._map._fadeAnimated) {
```

Comment:
opacity in <IE9 is never animated, so the second condition is superfluous, right?